### PR TITLE
Add the Poptip component

### PR DESCRIPTION
**Pull request by bzf**

### CODE REVIEW

File: src/mnd-bootstrap/components/_poptip.scss [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/*doc
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_poptip.scss]: Use `//` comments everywhere

Comment (bzf) [inline src/mnd-bootstrap/components/_poptip.scss]: But then the documentation doesn't work silly!

**Comment by bzf** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;UtbYEWF1&#x2F;1244-poptips-in-stories-where-to-find-insights"> Poptips in Stories where to find Insights</a>

File: src/mnd-bootstrap/components/_poptip.scss [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ Alerts should always starts with a word that tell the user more about the feedback in short.
+
+```html_example
+<div class="poptip">
+  <strong>POPTIP</strong>: This is a pretty dope tip!
+</div>
+```
+*/
+
+$poptip-background: #f9f9fb;
```
Comment (jorkas) [inline src/mnd-bootstrap/components/_poptip.scss]: Don't we have those grey colors somewhere as variables already? @bzf @Joelhelin If not maybe add to colors?

Or some of these ones that we should use instead?
colors

File: src/mnd-bootstrap/components/_poptip.scss [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+$poptip-border-color: #e0e3e9;
+$poptip-triangle-size: 10px;
+$poptip-triangle-right: 17px;
+$transparent: transparent;
+
+%arrow {
```
Comment (jorkas) [inline src/mnd-bootstrap/components/_poptip.scss]: Maybe we should rename the placeholder selector to `%poptype-arrow` or move it to some utilities placeholders/classes if it's more generic and can be used somewhere else?

**Comment by jorkas** [comment]
Great work, take a look at the comments and then it's ok from me 👍

**Pull request merged**